.gitignore

```diff
@@ -58,3 +58,4 @@ import-graph.txt
 import-graph.dot
 bin/rrweb-output
 model-manifest.json
+api-docs/dump.rdb
```

Comment:
can you make sure to remove this file (api-docs/dump.rdb)?

Reply:
needed to be removed from the filesystem. i am also confused as to how it made it into this folder, it seems redis related. wanna just remove the file and remove this entry from the .gitignore?